Explorer/Assets/StreamingAssets/Js/Modules/UserIdentity.js

```diff
@@ -1,5 +1,36 @@
 module.exports.getUserData = async function(message) {
-    const json = await UnityUserIdentityApi.GetOwnUserData();
-    const result = JSON.parse(json);    
-    return result;
+    const result = await UnityUserIdentityApi.GetOwnUserData();
+    const data = result.data;
+    
+    if (!data) {
+        return {};
+    }
+    
+    const avatar = data.avatar;
+    const wearables = [];
```

Comment:
This can be preallocated

Reply:
Can we really pre-allocate? We dont have control on what scenes can do with this array. Its not read-only